HIRS_Utils/src/main/java/hirs/tpm/eventlog/uefi/UefiBootOrder.java

```diff
@@ -0,0 +1,37 @@
+package hirs.tpm.eventlog.uefi;
+
+/**
+ * Class to process a UEFI BootOrder variable.
+ * UEFI spec version 2.8 section 3.3 on page 83 defines the Boot Order as:
+ *     an array of UINT16s that make up an ordered list of the Boot#### options.
+ */
+public class UefiBootOrder {
+  /** list of UINT16 Boot#### numbers.*/
+  private char[] bootOrder = null;
+
+  /**
+   * Process the BootOrder UEFI variable.
+   * @param order byte array holding the UEFI boot order variable.
+   */
+  UefiBootOrder(final byte[] order) {
+    bootOrder = new char[order.length / UefiConstants.SIZE_2];
+    for (int i = 0; i < order.length; i = i + UefiConstants.SIZE_2) {
```

Comment:
Please use += instead of i=i+int, just like you did below in the toString method

Reply:
OK